projects/plugins/jetpack/extensions/blocks/recipe/recipe-print.js

```diff
@@ -0,0 +1,87 @@
+const printBtns = document.querySelectorAll(
+	'.wp-block-jetpack-recipe-details__detail .wp-block-button__link'
+);
+const style = document.createElement( 'style' );
+
+printBtns.forEach( btn => {
```

Comment:
Would it be worth wrapping all this in a `domReady` from `@wordpress/dom-ready`, to ensure we do not run into any issues?

Reply:
Absolutely, this has been added!